shaman/models/nodes.py

```diff
@@ -8,8 +10,12 @@ class Node(Base):
     __tablename__ = 'nodes'
     id = Column(Integer, primary_key=True)
     host = Column(String(256), nullable=False, unique=True, index=True)
-    last_used = Column(DateTime, index=True)
+    # we use the old date to push new nodes to the top of the pool
+    default_last_used = datetime.datetime.now() - datetime.timedelta(days=365)
```

Comment:
Ok, I see now. This looks odd here as a class attribute. Maybe in a app level module like defaults.py?